### ISSUE TITLE
Doesn’t work on Android 13

### ISSUE THREAD

**Issue by Bnnzy** [ZaneYork/SMAPI-Android-Installer#312]

**Comment by lulateamo** [comment]
yes, I don't know how many times I tried to access the data folder. I hope that in the next update it will come back with the normal smapi

**Comment by Saucison78** [comment]
You use the fv file explorer but the key to get it to Work is to use the guide's version of the game to get the com.zane.stardewvalley folder after that you copy the files into the folder. The official game no longer works thanks to google. There will be no normal smapi as it was in 1.45 it's more complicated to get to work now since developers can no longer use custom directories. 

You probably noticed the stardewvalley folder is missing from the main storage where it used to be. That's google trying to crack down on piracy, all it does it restrict android gaming and it's modding capabilities.

I encourage any action that hurts google as a company, fuck them.

**Comment by lulateamo** [comment]
> Você usa o explorador de arquivos fv, mas a chave para fazê-lo funcionar é usar a versão do guia do jogo para obter a pasta com.zane.stardewvalley depois de copiar os arquivos para a pasta. O jogo oficial não funciona mais graças ao google. Não haverá smapi normal como era em 1.45, é mais complicado começar a trabalhar agora, pois os desenvolvedores não podem mais usar diretórios personalizados.
> 
> Você provavelmente notou que a pasta stardewvalley está faltando no armazenamento principal onde costumava estar. Isso é o Google tentando reprimir a pirataria, tudo o que ele faz é restringir os jogos do Android e seus recursos de modificação.
> 
> Encorajo qualquer ação que prejudique o Google como empresa, fodam-se eles.
> 
> Eu testei no android 13 a versão mais recente então funciona.

what guide?

**Comment by haru1727** [comment]
the alpha instalation guide, download the signed apk from the telegram link
https://github.com/ZaneYork/SMAPI-Android-Installer/issues/301#issue-1653459704

**Comment by BioDev28** [comment]
IDK if this will work on android 13 but if you go to storage/other files then it opens up a system explorer that has root access. i know it works on android 10.
